### PR TITLE
Bump dependencies, fix invalid access to fields, gh workflow

### PR DESCRIPTION
**Pull request by Bixilon**
As the commits say

### CODE REVIEW

File: src/main/kotlin/kr/heartpattern/exhibitionism/ExhibitionismTransformer.kt [hunk 1]
```diff
@@ -3,59 +3,65 @@ package kr.heartpattern.exhibitionism
...
     private fun transformAccess(type: Type, value: Int): Int {
         var access = value
         if (option.public) {
             access = (access and (ACC_PRIVATE or ACC_PROTECTED).inv()) or ACC_PUBLIC
         }
-        if (option.open && (type != Type.FIELD || !option.noStaticFinal || access and (ACC_FINAL and ACC_STATIC) != (ACC_FINAL and ACC_STATIC))) {
-            access = access and ACC_FINAL.inv()
+        if (option.open) {
+            val staticFinal = access and (ACC_FINAL or ACC_STATIC) == (ACC_FINAL or ACC_STATIC)
+            if (type != Type.FIELD || option.noStaticFinal || !staticFinal) {
```
Comment (HeartPattern) [inline src/main/kotlin/kr/heartpattern/exhibitionism/ExhibitionismTransformer.kt]:
```suggestion
            if (type != Type.FIELD || !option.noStaticFinal || !staticFinal) {
```

File: src/main/kotlin/kr/heartpattern/exhibitionism/App.kt [hunk 2]
```diff
@@ -11,14 +11,15 @@ import java.util.logging.Logger
...
+    private val input: File by option().file(mustExist = true).required()
+    private val output: File by option().file(mustExist = false).required()
     private val parallel: Int by option().int().default(Runtime.getRuntime().availableProcessors())
     private val nopublic: Boolean by option().flag()
     private val noopen: Boolean by option().flag()
     private val path: List<String> by option().split(",").defaultLazy { listOf() }
     private val noStaticFinal: Boolean by option().flag()
-    private val ignoreDuplicates: Boolean by option("--ignoreDuplicates").flag()
+    private val ignoreDuplicates: Boolean by option().flag()
+    private val fixInvalidAccess: Boolean by option().flag(default = true)
```
Comment (HeartPattern) [inline src/main/kotlin/kr/heartpattern/exhibitionism/App.kt:23]: readme.md should be updated too!

**Review by HeartPattern — CHANGES_REQUESTED** [review]

**Comment by Bixilon** [comment]
Absolutely right, fixed!

**Comment by Bixilon** [comment]
No wait, it should be the opposite. If we static final should be removed (`noStaticFinal`), we remove it. Otherwise we only remove it if it is not static final. right?

**Comment by HeartPattern** [comment]
Oh, naming is really confusing.. As README.md said, `noStaticFinal` option is "do not modify static final" field. Maybe we should rename it to `ignoreStaticFinal` or something.

**Pull request merged**